### DISCUSSION TITLE
Add display

### DISCUSSION THREAD

**Discussion by wimdm1** [jeroen79/esp-brew-engine#45] [Ideas]
Add a display (by using LVGL library expl. TFT 2,8" ILI9341 display) that shows the current temp / Output and Target temp / Output or even with a touchscreen so that a Start, Stop, Pauze buttons can be added. IP address would also be nice.

**Comment by jeroen79** [comment]
I personal use an old tablet to control the interface, its easier as i can place it where i want.

But i am not against trying to integrate a touchscreen, i can understand it easier in some setups, but as i do not have it lying around its quite difficult to support it atm.
